### ISSUE TITLE
Improve VoiceOver for hashtags

### ISSUE THREAD

**Issue by VyrCossont** [feditext/feditext#178]
**Is your feature request related to a problem? Please describe.**
It's annoying that "#foo" is pronounced "number foo" in VoiceOver for post and bio text.

**Describe the solution you'd like**
Hashtags should be pronounced "hashtag" or "hash" or "tag" or the "hash" part should be omitted, and which of those is used should be a config option. Also, they should be added to the text's custom action list along with regular links, and we should make sure they're not confused with regular or mention links.

We may also want to search the content DB for a version of the tag that isn't entirely lower-cased, if available, and use that text for the VoiceOver tag name, so that multi-word tags are not rendered unintelligible. See #173.

**Additional context**
Emoji might have similar problems; I'm not sure how they're currently handled.

See #177 for another case where we want to customize pronounciation.

Basically, we need an iOS equivalent of [`accessibilityCustomText`](https://developer.apple.com/documentation/foundation/nsattributedstring/key/2870144-accessibilitycustomtext), which [doesn't seem to exist](https://github.com/lionheart/openradar-mirror/issues/19652). So perhaps we generate an accessibility label for the post text that is the same as the text but with the hashtags spelled out.

[`accessibilityTextCustom`](https://developer.apple.com/documentation/foundation/nsattributedstring/key/2865778-accessibilitytextcustom) exists but the documentation is cryptic and it's unclear what it actually does, if anything: there is literally no Swift or Obj-C code on GitHub that uses it, so it's unlikely to be what we're looking for.
